redom/android/src/main/java/com/pushtorefresh/redom/android/recycler/TextViewViewHolder.kt

```diff
@@ -2,20 +2,53 @@ package com.pushtorefresh.redom.android.recycler
 
 import android.support.v7.widget.RecyclerView
 import android.widget.TextView
+import com.jakewharton.rxbinding2.view.RxView
+import com.jakewharton.rxbinding2.widget.RxTextView
+import com.jakewharton.rxrelay2.Relay
 import com.pushtorefresh.redom.android.view.TextViewImpl
 import com.pushtorefresh.redom.api.Component
 import io.reactivex.disposables.CompositeDisposable
+import io.reactivex.functions.Consumer
 import io.reactivex.rxkotlin.plusAssign
 
 class TextViewViewHolder(private val textView: TextView) : RecyclerView.ViewHolder(textView) {
     private val disposable = CompositeDisposable()
 
     fun bind(component: Component<*>) {
         disposable.clear()
-        disposable += component
-            .getChangeProperty<CharSequence>(TextViewImpl.C_PROPERTY_TEXT)
-            .subscribe { text ->
-                textView.text = text
-            }
+
+        component
+                .observeProperties
+                .entries
+                .forEach { property ->
+                    val name = property.key
+                    @Suppress("UNCHECKED_CAST") val relay = property.value as Relay<Any>
+
+                    val observable = when (name) {
+                        TextViewImpl.O_PROPERTY_TEXT_CHANGES -> RxTextView.textChanges(textView)
```

Comment:
looks like we can create observable only once cause it is lazy and it may be internal details - we have component and we call bind on it and it returns disposable or list of observables.